.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,64 @@
+name: "publish-npm"
+
+on:
+  release:
+    types: ["created"]
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  build:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@v4"
+        with:
+          submodules: "recursive"
+
+      - name: "Install task"
+        run: "npm install -g @go-task/cli"
+
+      - name: "Run package task"
+        run: "task package"
+
+      - name: "Upload build artifacts"
+        uses: "actions/upload-artifact@v4"
+        with:
+          path: "dist/"
+
+  publish-npm:
```

Comment:
How about `publish-to-npm`?

Reply:
Sounds good. Shall we also update the workflow `name` and filename to match this?

Reply:
Right, good idea.